CMakeLists.txt

```diff
@@ -76,12 +76,12 @@ endif()
 find_package(SWIG REQUIRED)
 include(${SWIG_USE_FILE})
 
-option(BUILD_PYTHON "Build Python SWIG module" OFF)
+option(BUILD_PYTHON "Build Python SWIG module" ON)
```

Comment:
Can you turn this off? We have Python explicitly enabled in the CI system, but OpenDB users might not all have the necessary python dependencies to build it (+not interested by default. e.g. Cherry).